dotnet/generate-target-platforms.csharp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env /Library/Frameworks/Mono.framework/Commands/csharp
```

Comment:
The generated files look like this https://gist.github.com/rolfbjarne/765c4e4b38b1b017380c9378d9666317